t/tomcat/build_info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "tomcat",
```

Comment:
we like to add apache packages like `apache-tomcat`. So no change need in build script. But change package_name here.

Also not build script and build_info.json should added in folder a/apche-tomcat

I see we already have folder also - https://github.com/ppc64le/build-scripts/tree/master/a/apache-tomcat.
Please make necessary changes